stock_picking_mgmt_weight/models/stock_move.py

```diff
@@ -0,0 +1,20 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    theoretical_qty = fields.Float(string='Theoretical quantity demanded')
+    tare = fields.Float(string='Tare')
+    date_tare_weight = fields.Date(string='Tare weight date')
+    gross_weight = fields.Float(string='Gross')
+    date_gross_weight = fields.Date(string='Gross weight date')
+    net_weight = fields.Float(string='Net', compute="_compute_net_weight")
+
+    def _compute_net_weight(self):
```

Comment:
For a computed, non-stored field in v13 every record must be calculated. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0 . If not, a cache miss error is thrown